Bottom: 2c525ddc363f90db35c19a25d7e6af403a762ef0
Top:    6e8967fe9825a56389158977f89787cd0f95dbe4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-06 02:36:40 +0100

Refresh of allow-polygons-with-clearlinef

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 50ca30d..dfe34b2 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -11,6 +11,7 @@
 #include "../hidint.h"
 #include "gui.h"
 #include "draw.h"
+#include "draw_funcs.h"
 #include "rtree.h"
 #include "polygon.h"
 #include "gui-pinout-preview.h"
@@ -1784,7 +1785,7 @@ ghid_draw_everything (BoxTypePtr drawn_area)
       gui->set_layer ("invisible", SL (INVISIBLE, 0), 0)) {
     if (PCB->ElementOn) {
       r_search (PCB->Data->name_tree[NAME_INDEX (PCB)], drawn_area, NULL, name_callback, &side);
-      draw_layer (&(PCB->Data->Layer[max_copper_layer + side]), drawn_area, NULL);
+      dapi->draw_layer (&(PCB->Data->Layer[max_copper_layer + side]), drawn_area, NULL);
     }
     if (global_view_2d) {
       r_search (PCB->Data->pad_tree, drawn_area, NULL, pad_callback, &side);
